### PR TITLE
Message polling

### PR DESCRIPTION
**Pull request by viviivanov**

### CODE REVIEW

File: src/lib/connection/connector/putMessage.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  if (connection) {
+    throw new Error('Connect called twice.');
+  }
+
+  const connectId = createPromise();
+  connection = getPromise(connectId).then(payload => sendMessage(dataStore(payload)));
+
+  window.Staffbase = window.Staffbase || {};
+  window.Staffbase.plugins = window.Staffbase.Staffbase || {};
+  window.Staffbase.plugins.getMessages = mutliMessageProvider;
```
Comment (Ninerian) [inline src/lib/connection/connector/putMessage.js]: Correct typo mutliMessageProvider -> multiMessageProvider

File: src/lib/connection/connection.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import postMessageLegacy from './connector/postmessage-legacy.js';
 import fallback from './connector/fallback.js';
 import postMessage from './connector/postmessage.js';
+import putMessage from './connector/putMessage.js';
 
-let connect = Promise.race([postMessage(), postMessageLegacy(), fallback()]);
+let connect = Promise.race([postMessage(), postMessageLegacy(), putMessage(), fallback()]);
```
Comment (Ninerian) [inline src/lib/connection/connection.js]: Maybe calling it messagePolling for better differencation.

**Comment by viviivanov** [comment]
Please merge the PR in my absence

File: src/lib/app.js [hunk 3]
```diff
@@ -74,14 +81,29 @@ export async function openLinkInternal(url) {
...
+ * Get all language info from the app.
+ *
+ * Shoud get split up in the future and
+ * removed from the public interface
+ *
+ * @return {Promise<any>}
+ */
+export async function getLanguageInfos() {
+  log.debug('app/getLanguageInfos');
+  return sendMessage(cmd.langInfos);
```
Comment (staust) [inline src/lib/app.js]: this seems to be the same as "getContentLnaguages". Also the commend seems strange to me

**Pull request merged**